examples/compose/gak.env.example

```diff
@@ -13,7 +13,7 @@ LISTEN=:301
 INTEGRATE_SSH=false
 LOG_LEVEL=debug
 LINUX_USER_ADD_TPL=adduser -D -s {shell} {username}
-LINUX_USER_ADD_WITH_GID_TPL=adduser -D -s {shell} -u {gid} {username}
+LINUX_USER_ADD_WITH_GID_TPL=adduser -D -s {shell} -g {gid} {username}
```

Comment:
This is actually not right for alpine linux.

```
BusyBox v1.30.1 (2019-06-12 17:51:55 UTC) multi-call binary.

Usage: adduser [OPTIONS] USER [GROUP]

Create new user, or add USER to GROUP

	-h DIR		Home directory
	-g GECOS	GECOS field
	-s SHELL	Login shell
	-G GRP		Group
	-S		Create a system user
	-D		Don't assign a password
	-H		Don't create home directory
	-u UID		User id
	-k SKEL		Skeleton directory (/etc/skel)
```

Reply:
I am not sure if `-G` works with `gid`. Can you please verify?

Reply:
`-g` is the GECOS field and has no impact on group assignment.
`-G` only works with group name not `GID`
`-u` assigns `gid` equal to `uid` when `-G` is not set.
In Alpine I have not found it possible to assign both `uid` and `gid` in one command.
Two commands are required for this
`addgroup --gid {gid} {groupname}` then
`adduser -D -s {shell} -u {uid} -G {groupname} {username}` 
To support this, we can split group creation and user creation and group assignment.
As it is `LINUX_USER_ADD_WITH_GID_TPL` will break for alpine with 2 commands.

Reply:
I'm no longer working on a project using this tool, so I don't have an environment set up to test these changes. I'll close this PR, since it seems this isn't the right fix.